### ISSUE TITLE
incompatibility with origins

### ISSUE THREAD

**Issue by Zdiamondwolf** [pog5/mc-leashplayers#5]
This mod seems like a lot of fun and I wish it worked with origins, because we want to use it on a server that hosts it, nothing happens when you try to lead others

**Comment by pog5** [comment]
Yeah I know about this, this is due to origins overriding the leashplayers mod's resources. I'll ty to get a fix out if i get some spare time.

**Comment by V-lpen** [comment]
looking forward to the fix of this, would like to use both together

**Comment by netcore10** [comment]
i second this

**Comment by V-lpen** [comment]
any hope of this bug being fixed?

**Comment by pog5** [comment]
Sadly with school starting, I don't think I'll have the time to write up a fix for this. Anyone who wants to fix it can open a pull request and I will more than gladly merge it! :)

**Comment by V-lpen** [comment]
any chance of this being fixed?

**Comment by pog5** [comment]
Closing this for now as Origins does not seem to be actively developed.
